### ISSUE TITLE
Reword "badges"

### ISSUE THREAD

**Issue by JocelynDelalande** [erming/shout#512]
In settings, we have a "Enable badge" option to enable desktop notifications.

Honestly, I had to test it to understand what does it means.

Is this "badge" word common te speak about "desktop notifications", or would you be ok rewording it with "desktop notifications" or "notifications" ?

**Comment by Xe** [comment]
it's the badge on the favicon.

**Comment by astorije** [comment]
Yes! I totally agree that we should reword `badge` into `notifications`. As a matter of fact, that's what it's called: http://www.w3.org/TR/notifications/

**Comment by JocelynDelalande** [comment]
> it's the badge on the favicon.

@Xe So this single option is doing two different things maybe… Because my browser asks me if I want to enable desktop notifications.

**Comment by floogulinc** [comment]
The setting appears to only toggle notifications. There is the code for changing the favicon, which works on Firefox but does not seem to work in Chrome, and is not toggled by the badge setting.

Much of the relevant code is [here](https://github.com/erming/shout/blob/master/client/js/shout.js#L571-L625).

There are actually two things called a badge, the [favico.js](http://lab.ejci.net/favico.js/) badges called by `favico.badge(string)` and the badges on the channel names. Neither are controlled by the "Enable badge" setting.
```js
favico.badge("!");
```
```js
var badge = button.find(".badge");
if (badge.length !== 0) {
	var i = (badge.data("count") || 0) + 1;
	badge.data("count", i);
	badge.html(i > 999 ? (i / 1000).toFixed(1) + "k" : i);
	if (highlight || isQuery) {
		badge.addClass("highlight");
	}
}
```
What the "Enable badge" setting does control, is browser notifications.
```js
if (settings.badge && Notification.permission == "granted") {
	var notify = new Notification(msg.from + " says:", {
		body: msg.text.trim(),
		icon: "/img/logo-64.png",
		tag: target
	});
	notify.onclick = function() {
		window.focus();
		button.click();
		this.close();
	};
	if (!isSafari) {
		window.setTimeout(function() {
			notify.close();
		}, 5 * 1000);
	}
}
```
Also, there is a setting with the variable `notification` but that is for the notification sound.
```js
if (settings.notification) {
	pop.play();
}
```
